modules/apps/frontend-js/frontend-js-react-web/npmscripts.config.js

```diff
@@ -14,20 +14,17 @@
 
 module.exports = {
 	build: {
-		bundler: {
-			exclude: {
-				lodash: false,
-			},
-			ignore: ['**/legacy/config.js'],
-		},
-		exports: {
-			'classnames': 'classnames/index',
-			'formik': 'formik/dist/index',
-			'prop-types': 'prop-types/index',
-			'react': 'react/index',
-			'react-dnd': 'react-dnd/dist/cjs/index',
-			'react-dnd-html5-backend': 'react-dnd-html5-backend/dist/cjs/index',
-			'react-dom': 'react-dom/index',
-		},
+		bundler: false,
+		tsc: false,
+		main: 'build/node/packageRunBuild/resources/js/index.js',
```

Comment:
This is just temporary for poc right?

Reply:
Yep. The last commit I've pushed changes it to poing to the src folder again :sweat_smile: